workspaces/npm/plugins/npm/src/index.ts

```diff
@@ -23,6 +23,8 @@
 export { npmPlugin } from './plugin';
 
 export { isNpmAvailable } from './utils/isNpmAvailable';
-export { NpmInfoCard } from './components/NpmInfoCard';
-export { NpmReleaseOverviewCard } from './components/NpmReleaseOverviewCard';
-export { NpmReleaseTableCard } from './components/NpmReleaseTableCard';
+export {
+  NpmInfoCard,
+  NpmReleaseOverviewCard,
+  NpmReleaseTableCard,
+} from './plugin';
```

Comment:
Can you please merge this with the export above?

Reply:
back to exporting from src folder now!